charts/public-service/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "service.name" -}}
```

Comment:
namespace should be `public-service` for the default namespace (same as chart name) it's also less confusing since "service" is also a k8s resource name

Reply:
5b05878